tools/oeedger8r/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "oeedger8r",
+  "version": "0.6.0",
```

Comment:
If we update our version number (which we will do to 0.7.0 soon) I'd like this to not get left behind. Maybe we should make a comment in the RELEASING.md document that tells the release manager to update this file?